docs/integration/opentelemetry.md

```diff
@@ -0,0 +1,35 @@
+
+This page explains how to integrate OpenTelemetry metrics and logs in your application and how to use the [Sematext Agent](/docs/agents/sematext-agent) to ship logs and metrics from the application that's been instrumented with OpenTelemetry to Sematext Cloud. With the support for [OpenTelemetry Tracing](/docs/tracing/), you will be collecting metrics, logs, and traces in a single platform, giving you the ability to correlate them easily and understand how metrics trends, log events, and traces relate to each other across your applications.
+
+With logs, you can monitor your applications and infrastructure, troubleshoot issues faster, detect anomalies, and connect logs to [events](/docs/events) such as deployments or restarts. Metrics provide insights into system and application performance, while traces let you analyze request flows and latency across services. Together, they give you the full picture in one platform. Features like [Split Screen](/docs/guide/split-screen) and [Connected Apps](/docs/guide/connected-apps) in Sematext Cloud let you view and correlate this data side by side for faster and deeper root-cause analysis.
+
+## Why use OpenTelemetry with Sematext Cloud?
+
+OpenTelemetry provides a standard way to instrument applications, and Sematext Cloud lets you store, visualize, and alert on the telemetry data they produce.
+
+With OpenTelemetry Logs and Metrics in Sematext Cloud, you can:
+
+- Monitor your **applications as first-class entities**, not just the infrastructure they run on
+- Track **application-level performance** such as request rates, latency, errors, and custom business metrics
+- **Alert on application behavior**, not just resource usage
+- **Correlate logs, metrics, and traces** to quickly understand what happened and why
+- Connect telemetry with **events** like deployments or restarts to spot regressions faster
+
+Below, you’ll find step-by-step instructions for setting up OpenTelemetry and configuring your applications to ship logs and metrics.
+ 
+## Install Sematext Agent
+
+- Create an OpenTelemetry Logs or Monitoring [App](/docs/guide/app-guide). This will let you install the agent and control access to your monitoring and logs data.
+- Set up log and metrics exporting in your application. Follow the SDK-specific instructions provided in the UI. Choose your language and integrate OpenTelemetry metrics and logging into your code.
+- Install the [Sematext Agent](/docs/agents/sematext-agent/releasenotes) using the instructions shown in the App creation flow. The Agent works as an OpenTelemetry collector and ships logs from your application that has been instrumented with an OpenTelemetry SDKs to Sematext Cloud.
+
+OpenTelemetry metrics and logging is supported in `Python`, `Node.js`, `.NET`, `Java`. The SDK setup instructions for these languages **are shown after you create an OpenTelemetry Logs or Monitoring App**.
```

Comment:
I thought I also saw Ruby, PHP, Go... no?

Reply:
After @cpipilas went through each of these, he updated them, and these four are the ones that currently have the most straightforward approach with full OpenTelemetry support. I think we'll add the rest gradually as we see how support evolves on the OpenTelemetry side. For now, the repo that we are planning to make public contains these four: https://github.com/sematext/sematext-opentelemetry-examples/tree/main